Luminary099/R31.agc

```diff
@@ -212,11 +212,11 @@ GETRVN		CA	PRIO22		# INHIBIT SERVICER
 		TC	PRIOCHNG
 		TC	INTPRET
 		VLOAD	SETPD
-			RN		# LM STATE VECTOR IN RN,VN.
+			RN		# LM STATE VECTOR IN RN.VN
 			0
 		STOVL	RONE
 			VN
-		STOVL	VONE		# LOAD R(CSM),V(CSM) IN CASE MUNFLAG SET
+		STOVL	VONE		# LOAD R(CSM).V(CSM) IN CASE MUNFLAG SET
 			V(CSM)		# (TO INSURE TIME COMPATIBILITY)
```

Comment:
Typo in source (image snippet of scan won't upload unfortunately, line `0169` of page `706`)

```suggestion
			V(CSM)		# (TO INSURE TIME COMPATABILITY)
```